hazelcast-spring/src/main/resources/hazelcast-spring-4.0.xsd

```diff
@@ -4149,40 +4150,29 @@
         </xs:sequence>
     </xs:complexType>
 
-    <xs:simpleType name="wan-ack-type-format-enum">
```

Comment:
Shouldn't we keep this (with `non-space-string` `base`) to keep the enumeration part of the validation?

Reply:
Looks like this enum was moved from a XML node to the properties list so I don't know if we can enforce the enum in any way.

Reply:
Yeah, indeed, thanks. So it's a leftover to remove.